### DISCUSSION TITLE
Про шрифты

### DISCUSSION THREAD

**Discussion by ninedev-i** [razrabs-media/journal#359] [Polls]
Есть ощущение, что такой шрифт на темном фоне тяжело читается (из-за жирности начертания) - на белом намного лучше (а я люблю темные темы!). 
Также, может стоит уменьшить межстрочное расстояние со 140 до 130.

Может поиграемся со шрифтами?) Насколько вы открыты к предложениям/изменениям?

**Comment by philippranzhin** [comment]
Шрифт красивый, и потому мы готовы терпеть низкую читаемость, но в деталях вроде интервалов и отступов дизайнер уже работает

**Comment by batyshkaLenin** [comment]
Для меня дело не в шрифте, а в том, что я открывая статью на ПК смотрю вбок, хотя фактически, мне не нужно плавающее превью слева, картинка в превью не для того, чтобы я на неё постоянно глазел, я увидел картинку и заголовок и пошёл читать текст, а при чтении текста и смотрю вбок, на телефоне удобнее читать.
